### PR TITLE
NG 13

### PR DESCRIPTION
**Pull request by ngohungphuc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,30 +43,29 @@
...
-    "rxjs": "~7.4.0",
-    "tslib": "^2.0.0",
-    "zone.js": "~0.11.4"
+    "rxjs": "~7.5.6",
+    "zone.js": "~0.11.8"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^12.2.16",
+    "@angular-devkit/build-angular": "^13.3.9",
     "@angular-eslint/builder": "12.7.0",
```
Comment (ngohungphuc) [inline package.json:60]: Need to bump these version to 13 too
```
    "@angular-eslint/builder": "12.7.0",
    "@angular-eslint/eslint-plugin": "12.7.0",
    "@angular-eslint/eslint-plugin-template": "12.7.0",
    "@angular-eslint/schematics": "12.7.0",
    "@angular-eslint/template-parser": "12.7.0",
```

File: package.json [hunk 2]
```diff
@@ -16,80 +15,82 @@
     "e2e": "npx playwright test",
     "lint": "nx run-many --all --target=lint",
     "format": "prettier --write \"{apps,libs}/**/*.{ts,md,html}\" \"!{apps,libs}/**/coverage\" ",
-    "format:check": "prettier --list-different \"{apps,libs}/**/*.{ts,md}\" \"!{apps,libs}/**/coverage\""
+    "format:check": "prettier --list-different \"{apps,libs}/**/*.{ts,md}\" \"!{apps,libs}/**/coverage\"",
+    "check-update": "ng update && ncu -u",
```
Comment (santam85) [inline package.json:19]: Can we remove these 2 lines? It's an opinionated pattern not strictly needed for the update

**Comment by santam85** [comment]
This seems to be missing some tsconfig changes the Angular schematics should be performing?

**Comment by ngohungphuc** [comment]
@santam85 I used the ng update and follow the steps from update.angular.io

**Comment by ngohungphuc** [comment]
@santam85 Can you check it again. Thank you

**Pull request merged**